wire/internal/wire/wire_test.go

```diff
@@ -494,7 +498,7 @@ func loadTestCase(root string, wireGoSrc []byte) (*testCase, error) {
 		if !info.Mode().IsRegular() || filepath.Ext(src) != ".go" {
 			return nil
 		}
-		data, err := ioutil.ReadFile(src)
+		data, err := readFile(src)
```

Comment:
Not sure if this one needs to be converted...?

Reply:
(Same advice here as the one above.)